### PR TITLE
Create Bresenham-Circle.c

### PR DESCRIPTION
**Pull request by Utkarsh2604**
# Title/Heading

Bresenham Circle Drawing Algorithm using C.

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generates no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Utkarsh2604** [comment]
please merge my pr

File: C/Bresenham-Circle.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#include <stdio.h>
```
Comment (kaustubhgupta) [inline C/Bresenham-Circle.c:1]: Please add some documentation in the form of comments to explain what the Bresenham Algorithm is all about, why it is faster than the midpoint algorithm

File: C/Bresenham-Circle.c [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+   }
+
+    void BresenhamCircle(int xc,int yc,int r)
+   {
+    int x=0,y=r,d=3-(2*r);
+    Circle(xc,yc,x,y);
+
+    while(x<=y)
+     {
+      if(d<=0)
```
Comment (kaustubhgupta) [inline C/Bresenham-Circle.c:24]: use better variable names

File: C/Bresenham-Circle.c [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+       printf("Enter the value of radius  :");
+       scanf("%d",&r);
+       BresenhamCircle(xc,yc,r);
+       BresenhamCircle(xc+r,yc,r);
+       BresenhamCircle(xc-r,yc,r);
+       BresenhamCircle(xc,yc+r,r);
+       BresenhamCircle(xc,yc-r,r);
+     getch();
+     closegraph();
+     return 0;
```
Comment (kaustubhgupta) [inline C/Bresenham-Circle.c:57]: added a screenshot of the program and put both files (code and output) in one folder under the C directory

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@Utkarsh2604 are you working on this ?

**Pull request merged**